epictrack-web/src/utils/exportUtils.ts

```diff
@@ -0,0 +1,37 @@
+import { json2csv } from "json-2-csv";
+import dateUtils from "./dateUtils";
+import { MRT_TableInstance } from "material-react-table";
+import { MRT_RowData } from "material-react-table";
+interface ExportToCsvOptions<T extends MRT_RowData> {
+  table: MRT_TableInstance<T>;
+  downloadDate: string | null;
+  filenamePrefix: string;
+}
+
+export async function exportToCsv<T extends MRT_RowData>({
```

Comment:
This seems to be specific to the material react table.

let's either:
- Make it more general so that any component can use it
- Place it under the same folder where the track table is